packages/itwinui-css/src/tabs/base.scss

```diff
@@ -130,6 +130,7 @@ $borderless-horizontal-tab-min-height: calc(
   grid-template-columns: 1fr auto;
   grid-template-rows: auto 1fr;
   contain: inline-size;
+  inline-size: 100%;
```

Comment:
You can undo #1911 now

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1955/commits/7f88dcd263e6519b38730bca28f5d150a2ca0483)